scripts/tools/combine-tools.js

```diff
@@ -25,9 +24,14 @@ const fuse = new Fuse(list, options)
 
 const getFinalTool = async (toolObject) => {
     let finalObject = toolObject;
-    const languageSearch = await fuse.search(toolObject.filters.language)
-    if (languageSearch.length) {
-        finalObject.filters.language = languageSearch[0].item;
+
+    //there might be a tool without language
```

Comment:
in case of CLIs there is no language

Reply:
Means that there can be certain tools in list whose language is not defined, right?